src/main/java/org/elasticsearch/index/engine/EngineConfig.java

```diff
@@ -391,4 +388,27 @@ public void setGcDeletesInMillis(long gcDeletesInMillis) {
     public void setCompoundOnFlush(boolean compoundOnFlush) {
         this.compoundOnFlush = compoundOnFlush;
     }
+
+    /**
+     * Returns the {@link org.elasticsearch.index.shard.TranslogRecoveryPerformer} for this engine. This class is used
+     * to apply transaction log operations to the engine. It encapsulates all the logic to transfer the translog entry into
+     * an indexing operation.
+     */
+    public TranslogRecoveryPerformer getTranslogRecoveryPerformer() {
+        return tanslogRecoveryPerformer;
+    }
+
+    /**
+     * If set to <code>true</code> initial translog recovery is skipped.
+     */
+    public void setSkipInitialRecovery(boolean skipInitialRecovery) {
+        this.skipInitialRecovery = skipInitialRecovery;
+    }
+
+    /**
+     * Returns <code>true</code> if initial translog recovery should be skipped. Otherwise <code>false</code>
+     */
+    public boolean isSkipInitialRecovery() {
```

Comment:
I know the boolean pattern uses "is" but I think this would make more sense named `shouldSkipInitialRecovery`?